src/widgetComponents/Matrix1dInput/index.js

```diff
@@ -63,27 +74,52 @@ export default class Matrix1dInput extends React.PureComponent {
     }
 
     // TODO: memoize this later
-    rendererParams = (key, row) => ({
-        title: row.title,
-        tooltip: row.tooltip,
-        cells: row.cells,
-        // FIXME: send rowKey and cellKey as props
-        onCellClick: cellKey => this.handleCellClick(key, cellKey),
-        onCellDrop: (cellKey, droppedData) => this.handleCellDrop(key, cellKey, droppedData),
-        selectedCells: this.props.value ? this.props.value[key] : undefined,
-        disabled: this.props.disabled,
-        readOnly: this.props.readOnly,
-    })
+    rendererParams = (key, row) => {
+        const {
+            value,
+            disabled,
+            meta: {
+                orientation = 'horizontal',
+            },
+            readOnly,
+        } = this.props;
+
+        const {
+            title,
+            tooltip,
+            cells,
+        } = row;
+
+        return ({
+            title,
+            tooltip,
+            cells,
+            orientation,
+            // FIXME: send rowKey and cellKey as props
+            onCellClick: cellKey => this.handleCellClick(key, cellKey),
+            onCellDrop: (cellKey, droppedData) => this.handleCellDrop(key, cellKey, droppedData),
+            selectedCells: value ? value[key] : undefined,
+            disabled,
+            readOnly,
+        });
+    }
 
     render() {
         const {
             options,
+            meta: {
+                orientation = 'horizontal',
```

Comment:
Setting default value for `orientation` in multiple points can silently break in future changes